### DISCUSSION TITLE
Disable Select Control

### DISCUSSION THREAD

**Discussion by getsamonnet** [JedWatson/react-select#4490] [Q&A] [answered]
How to disable the full Select Control on a button click? Looks like Disabled prop is not working

**Comment by ebonow** [comment, marked as answer]
Greetings @getsamonnet ,

Have you tried `isDisabled`?

>**isDisabled** _boolean = false_
>Is the select disabled

**Reply by getsamonnet** [reply]
Hi @ebonow 

Yes tried, didn't disable the Select control.. Isn't isDisabled used only for Options?

**Reply by ebonow** [reply]
@getsamonnet `isDisabled` prop is used to disable the Select
[demo](https://codesandbox.io/s/react-select-isdisabled-ucf7f)

`isOptionDisabled` is used as a function to disable options

**Reply by getsamonnet** [reply]
@ebonow Thanks for the demo,  my mistake, used a wrong case. :) It worked fine now... Thanks again for quick response...
